src/utils/viewer/viewer.cpp

```diff
@@ -931,9 +954,6 @@ void Viewer::drawEvent() {
       Mn::GL::Renderer::disable(Mn::GL::Renderer::Feature::Blending);
     }
     sensorRenderTarget->blitRgbaToDefault();
-    // Immediately bind the main buffer back so that the "imgui" below can work
-    // properly
-    Mn::GL::defaultFramebuffer.bind();
```

Comment:
Why is this removed? What's the reason?

Reply:
Good catch. I am supposed to move it out of the bracket. Definitely a bug introduced. Will fix.